OpenRA.Mods.Common/Traits/Mobile.cs

```diff
@@ -506,6 +564,11 @@ public bool CanEnterCell(CPos cell, Actor ignoreActor = null, bool checkTransien
 			return Info.CanEnterCell(self.World, self, cell, ignoreActor, checkTransientActors ? CellConditions.All : CellConditions.BlockedByMovers);
 		}
 
+		public bool CanMoveFreely(CPos cell, Actor ignoreActor = null, bool checkTransientActors = true)
```

Comment:
For the time being, for this PR I'll name it as "CanMoveFreelyInto" refering whether the actor can move without problems in a navigable cell. In a next PR I'm working of a heavy refactor of the MobileInfo Trait, this function will get called "CollidesWithOtherActorsInCell", but this naming forces me to change (basically by negating) the way callers work with this function.

Allow me to name it as I have just told you for the time being.